### ISSUE TITLE
if i use gdb-peda it will be waiting for debugger

### ISSUE THREAD

**Issue by nigacat423** [Gallopsled/pwntools#1330]
# Pwntools Issue Template

Thanks for contributing to Pwntools!

When reporting an issue, be sure that you are running the latest released version of pwntools (`pip install --upgrade pwntools`).

Please verify that your issue occurs on 64-bit Ubuntu 14.04.  You can use the Dockerfile on `docker.io` for quick testing.

```
$ docker pull pwntools/pwntools:stable
$ docker run -it pwntools/pwntools:stable
```

If possible, provide a proof-of-concept which demonstrates the problem.  Include any binaries or scripts necessary to reproduce the issue, and please include the full debug output via setting the environment variable `PWNLIB_DEBUG=1`.
![image](https://user-images.githubusercontent.com/49108901/62449778-845dd080-b79d-11e9-8b73-e0a8f64de16e.png)

**Comment by mauricesvp** [comment]
See #1016
When calling process("./pwnme_fastbin"), the binary gets executed, and probably is finished/exited before gdb can attach to it.
You probably want to use gdb.debug() instead.
If there's still a problem please include your (entire) script so that we have all the information.

**Comment by nigacat423** [comment]
thanks i have solved this problem!

**Comment by Arusekk** [comment]
A similar issue fixed in d8b69dde24b0fc34d72f61d492489487cac28418
